terraform/src/common/container_apps.tf

```diff
@@ -1,10 +1,11 @@
 module "aca_wagtail" {
-  source = "git::https://github.com/nhsuk/dct.terraform-modules.wagtail-container-apps?ref=0.4.1"
+  source = "git::https://github.com/nhsuk/dct.terraform-modules.wagtail-container-apps?ref=98ad755b76a0447f52d1d3e5f8eadc441987012a"
```

Comment:
Update to tag after https://github.com/nhsuk/dct.terraform-modules.wagtail-container-apps/pull/9 is merged and tagged